table.rst

```diff
@@ -0,0 +1,43 @@
+
+Table for astype
+
+
++-------------------+-------------------------+
+|  S1               |  Dtype                  |
++===================+=========================+
+|  int              |  Int8Dtype,Int16Dtype,  |
+|                   |  Int32Dtype,Int54Dtype, |
++-------------------+-------------------------+
+|  str              | StringDtype,str         |
++-------------------+-------------------------+
+|  bytes            | BolleanDtype            |
++-------------------+-------------------------+
+|  float            |Float32Dtype,Float64Dtype|
++-------------------+-------------------------+
+|  complex          |                         |
++-------------------+-------------------------+
+|  Timestamp        |  Timestamp              |
++-------------------+-------------------------+
+|  Timedelta        | Timedelta,TimedeltaIndex|
```

Comment:
Same comment as above - can't pass those arguments to `astype()`, and I think we can remove `Timedelta` from `S1`, because we have `TimedeltaSeries`.